api/permissioned.go

```diff
@@ -77,7 +77,7 @@ func permissionedAny(in interface{}, out interface{}) {
 			}
 		}
 		if !ok {
-			panic("unknown 'perm' tag on " + field.Name)
+			panic("unknown 'perm' tag on " + field.Name) // is this okay? can this be used to crash the process?
```

Comment:
Same here.

Reply:
Yep, those 2 are just sanity checks for perm tags on the api struct